### ISSUE TITLE
Removed from Chrome Webstore

### ISSUE THREAD

**Issue by nomiDesigns** [RemLampa/facebook-video-speed-control#28]
It has been removed from the Chrome Webstore. Where else I can find it?

**Comment by RemLampa** [comment]
@nomiDesigns Yeah, unfortunately it was taken down. :cry: It got a copyright complaint from Facebook for usage of the "Facebook" name and likeness of their logo.
